code/game/machinery/computer/station_alert.dm

```diff
@@ -68,10 +71,13 @@
 				L -= I
 	return !cleared
 
-
 /obj/machinery/computer/station_alert/process()
+	air_alarm_count = LAZYLEN(alarms["Atmosphere"])

```

Comment:
why are you storing these every process loop?
Ok so, these ```X_count``` vars are on the ```station_alert``` console, which is where the ```alarms``` var is, meaning they're both still accessibly from the same place, your vars are completely unneeded, just do the ```LAZYLEN()``` operations wherever you previously use the ```X_count``` vars.